dot-base/fhir-server
# feat: disable cascading delete

# Ticket 🎫

This closes #245.

BREAKING CHANGE: Cascading delete is disabled. Requests relying on this feature will no longer succeed.

# Description 📖
This PR updates the config file `application.yml` to deactivate the cascading delete.

# How to Test? 🧪

Send a delete request for a resource with dependencies and the header `X-Cascade: "delete"`. It should fail and report that there were dependencies and cascading deletes are disabled.

# References (optional) 🔗

https://smilecdr.com/docs/fhir_repository/deleting_data.html

# Checklist ✅
